test/hashly.unittests.js

```diff
@@ -258,20 +261,44 @@ describe("hashly", function () {
             assert.equal(exitCode, 0);
         });
 
-        it("should return -1 and log an error if createManifest throws an exception", function () {
+        it("should check for sourcemap file and modify the js file with the hashed map file name", function() {

```

Comment:
The error with this test was it was trying to write to /alt/b which I couldn't create on travis but could on docker and my own box. A better solution besides using relative path is to override the part where it writes the manifest. I don't think it matters a whole lot.